sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -368,7 +368,7 @@ def analyze_callback(raw_response, _, headers):  # pylint: disable=unused-argume
             return prepare_form_result(analyze_result, model_id)
 
         deserialization_callback = cls if cls else analyze_callback
-        return self._client.begin_analyze_with_custom_model(
+        return self._client.begin_analyze_with_custom_model(  # type: ignore
```

Comment:
Why type ignore?

Reply:
generated code now expects LROPoller[None] and we return LROPoller[List[RecognizedForm]]. Talked to @iscai-msft about this already, she said to ignore. it would need to be updated in the swagger for the gen code to have the correct type. Not sure about how to do that since this is a custom type.

Reply:
interesting!